components/PageControl/src/MDCPageControl.m

```diff
@@ -111,22 +113,16 @@ - (void)layoutSubviews {
     return;
   }
   self.hidden = NO;
-  for (MDCPageControlIndicator *indicator in _indicators) {
-    NSInteger indicatorIndex = [_indicators indexOfObject:indicator];
-    if (indicatorIndex == _currentPage) {
+
+  for (NSUInteger pageNumber = 0; pageNumber < _indicators.count; pageNumber++) {
```

Comment:
Consider:
for (MDCPageControlIndicator *indicator in _indicators) {
}

Reply:
I'm using `pageNumber` in line 119. With that in mind, are you okay with not using fast enumeration here?